frontend/public/features.ts

```diff
@@ -228,6 +230,9 @@ const detectShowOpenShiftStartGuide = (dispatch, canListNS: boolean = false) =>
 
 // Check the user's access to some resources.
 const ssarChecks = [{
+  flag: FLAGS.CAN_GET_NS,
```

Comment:
Probably we should have used this for the `requirePrometheus` HOC, but the RBAC check in that component will be going away in #909 regardless.

Reply:
👍 Yes, I think `requirePrometheus` should have been using `FLAGS.CAN_GET_NS` instead of `FLAGS.CAN_LIST_NS`, but like you say, it doesn't matter now.